src/commands/Fun/Random/poll.ts

```diff
@@ -0,0 +1,38 @@
+import { SteveCommand } from '@lib/structures/commands/SteveCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { Message } from 'discord.js';
+import { NAME } from '@root/config';
+
+export default class extends SteveCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['vote'],
+			description: `Have ${NAME} create a poll for you`,
+			extendedHelp: 'This command requires you to have at least two options and no more than ten.',
+			examples: ['poll Is butt legs?|Butt is legs|Butt is butt'],
+			usage: '<question:string> <option1:string> <option2:string> [...]',
+			helpUsage: 'question | option 1 | option 2 | ...'
+		});
+	}
```

Comment:
the description and extended help need to be internationalized, and helpUsage is unnecessary now. you can look at other commands and the en-US.ts language file to see how that should be done